src/data/source/Benchmark DataExplainer.md

```diff
@@ -23,7 +23,7 @@ unit information - here's a table of the mapping so you can see the units clearl
 : Water Use (kGal)                              : WaterUse                      :
 : ENERGY STAR Score                             : ENERGYSTARScore               :
 : Electricity Use (kBtu)                        : ElectricityUse                :
-: Natural Gas Use (kBtu)                        : NaturalGasUse                 :
```

Comment:
I actually would not recommend changing this - we don't need to rename the data in our data pipeline, because that'll make it harder for folks to compare parsed data to the raw city data. Could you revert any changes that tweak the data?